This is not passing it's own tests.

```
[ERROR] Failures: 
[ERROR]   ShutdownMonitorTest.testNoExitSystemProperty:142 expected: <false> but was: <true>
```

--- a/jetty-server/src/test/java/org/eclipse/jetty/server/ShutdownMonitorTest.java
+++ b/jetty-server/src/test/java/org/eclipse/jetty/server/ShutdownMonitorTest.java
@@ -133,6 +133,16 @@ public class ShutdownMonitorTest
         assertTrue(!monitor.isAlive());
     }
 
+    /*
+     * Disable these config tests because ShutdownMonitor is a 
+     * static singleton that cannot be unset, and thus would
+     * need each of these methods executed it its own jvm -
+     * current surefire settings only fork for a single test 
+     * class.
+     * 
+     * Undisable to test individually as needed.
+     */
+    @Disabled
     @Test
     public void testNoExitSystemProperty() throws Exception
     {
@@ -164,16 +174,7 @@ public class ShutdownMonitorTest
             assertTrue(!ShutdownMonitor.isRegistered(server));
         }
     }
-
-    /*
-     * Disable these config tests because ShutdownMonitor is a 
-     * static singleton that cannot be unset, and thus would
-     * need each of these methods executed it its own jvm -
-     * current surefire settings only fork for a single test 
-     * class.
-     * 
-     * Undisable to test individually as needed.
-     */
+    
     @Disabled
     @Test
     public void testExitVmDefault() throws Exception

